dags/bq_events_to_amplitude.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+
+from airflow import models
+from utils.gcp import bigquery_etl_query
+from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
+from airflow.contrib.operators.gcs_to_s3 import GoogleCloudStorageToS3Operator
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'start_date': datetime.datetime(2019, 6, 27),
+    'email': ['telemetry-alerts@mozilla.com', 'frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'depends_on_past': False,
+    'retries': 2,
+    'retry_delay': datetime.timedelta(minutes=10),
+}
+
+dag_name = 'bq_events_to_amplitude'
+
+# This connection needs access to both BQ and GCS
+gcp_conn_id = 'google_cloud_derived_datasets'
```

Comment:
We may want to make a separate gcp connection, with a different project, to use for the GCS access. That way we don't need to change the `google_cloud_derived_datasets` service account to use GCS.